core/templates/dev/head/components/summary-tile/story-summary-tile.directive.ts

```diff
@@ -36,7 +36,7 @@ angular.module('oppia').directive('storySummaryTile', [
           var ctrl = this;
           ctrl.getStoryLink = function() {
             return UrlInterpolationService.getStoryUrl(
-              '/story', ctrl.getStoryId());
+              '/' + ctrl.getStoryId());
```

Comment:
Do you need the '/'? It seems to me that a storyId would be the logical argument to this method.

Reply:
So the function does a validation on the parameter, and that validation expects there to be . a '/' symbol that's why I added it. I didn't want to change the logic of getStoryUrl() , the other option is to create another function in UrlInterpolationService that doesn't require this validation. What do you suggest @seanlip ?

Reply:
I believe @aks681 has a fix for this so I am reverting this change. This PR now will only fix the pretest issue